src/descriptors/hydro_power_system_structs.json

```diff
@@ -173,10 +173,10 @@
                 },
                 {
                     "name": "upstream",
-                    "comment": "Upstream units",
-                    "null_value": "Vector{PSY.HydroGen}()",
-                    "default": "Vector{PSY.HydroGen}()",
-                    "data_type": "Vector{PSY.HydroGen}"
+                    "comment": "Upstream units, water lag, energy multiplier",
```

Comment:
We should be clear here about what each of these values mean. Specifically the lag and multiplier.
- lag: duration in number of periods between upstream release and downstream availability
- multiplier: relationship between upstream energy release and downstream energy availability
 ... or something along those lines